desktop/components/commercial_filter/filters/keyword/index.jade

```diff
@@ -0,0 +1,13 @@
+#keyword-search-bar-container.typeahead
+  input#keyword-search-bar-input.bordered-input(
+    placeholder='Search…'
+    tabindex='1'
+    itemprop='query-input'
+    type='text'
+    name='keyword'
+    value= term
```

Comment:
Oh you're right, I'll try to set the `term` local var as the keyword... might allow me to remove some code in the view that currently sets that manually from the url param.